src/Symfony/Bundle/TwigBundle/DependencyInjection/Compiler/ExtensionPass.php

```diff
@@ -63,13 +63,12 @@ public function process(ContainerBuilder $container)
             $container->getDefinition('twig.extension.code')->replaceArgument(0, $container->getParameter('templating.helper.code.file_link_format'));
         }
 
+        if ($container->getParameter('kernel.debug') && $container->has('debug.stopwatch')) {
```

Comment:
should it really be tied to the presence of ``debug.stopwatch`` (which is optional in the service definition already) ? this would prevent getting the Twig profiling data when you don't have the Stopwatch